sdk/eventgrid/azure-messaging-eventgrid/pom.xml

```diff
@@ -62,7 +62,7 @@
     <dependency>
       <groupId>com.azure</groupId>
       <artifactId>azure-core</artifactId>
-      <version>1.14.0</version> <!-- {x-version-update;unreleased_com.azure:azure-core;dependency} -->
```

Comment:
Why was this using an unreleased tag since 1.14.0 is already released?

Reply:
This is a hotfix branch based on the release SHA for `1.14.0` 😃